src/JSInterop/Microsoft.JSInterop/src/JSRuntimeBase.cs

```diff
@@ -124,35 +124,37 @@ private async Task<T> InvokeWithDefaultCancellation<T>(string identifier, IEnume
         protected abstract void BeginInvokeJS(long taskId, string identifier, string argsJson);
 
         /// <summary>
-        /// Allows derived classes to configure the information about an exception in a JS interop call that gets sent to JavaScript.
+        /// Completes an async JS interop call from JavaScript to DotNet.
         /// </summary>
-        /// <remarks>
-        /// This callback can be used in remote JS interop scenarios to sanitize exceptions that happen on the server to avoid disclosing
-        /// sensitive information to remote browser clients.
-        /// </remarks>
-        /// <param name="exception">The exception that occurred.</param>
-        /// <param name="assemblyName">The assembly for the invoked .NET method.</param>
-        /// <param name="methodIdentifier">The identifier for the invoked .NET method.</param>
-        /// <returns>An object containing information about the exception.</returns>
-        protected virtual object OnDotNetInvocationException(Exception exception, string assemblyName, string methodIdentifier) => exception.ToString();
```

Comment:
I decided to change the extensibility because it made logging harder and had some missing pieces of data `(dotNetObjectId)`